tests/connect/standalone-ssl-0004.phpt

```diff
@@ -43,8 +38,8 @@ printf("Inserted: %d\n", $retval->getInsertedCount());
 ===DONE===
 <?php exit(0); ?>
 --EXPECTF--
-OK: Got MongoDB\Driver\Exception\SSLConnectionException thrown from executeBulkWrite
-%s
+OK: Got MongoDB\Driver\Exception\ConnectionTimeoutException thrown from executeBulkWrite
+No suitable servers found (`serverSelectionTryOnce` set): [%s calling ismaster on '%s:%d']
```

Comment:
SSLConnectionException will no longer be used, since libmongoc only exposes SSL errors through server selection failures (i.e. ConnectionTimeoutException). This is a regrettable BC break for folks that were catching SSLConnectionException, but it's outside of our control.

I propose we document that SSLConnectionException is no longer useful and only applied to PHPC < 1.2.0.